doc/configtables/sector.csv

```diff
@@ -5,9 +5,17 @@ biomass,--,"{true, false}",Flag to include biomass sector.
 industry,--,"{true, false}",Flag to include industry sector.
 agriculture,--,"{true, false}",Flag to include agriculture sector.
 district_heating,--,,`prepare_sector_network.py <https://github.com/PyPSA/pypsa-eur-sec/blob/master/scripts/prepare_sector_network.py>`_
--- potential,--,float,maximum fraction of urban demand which can be supplied by district heating. Ignored where below current fraction.
--- progress,--,Dictionary with planning horizons as keys., Increase of today's district heating demand to potential maximum district heating share. Progress = 0 means today's district heating share. Progress = 1 means maximum fraction of urban demand is supplied by district heating
--- district_heating_loss,--,float,Share increase in district heat demand in urban central due to heat losses
+#NAME?,--,float,maximum fraction of urban demand which can be supplied by district heating. Ignored where below current fraction.

```

Comment:
@amosschle reviewing now, something went wrong with the configtables (`#Name?`). Could you revert this?

Reply:
My bad, thanks for spotting thiis. Should be fixed now.